dpnp/tests/test_histogram.py

```diff
@@ -101,6 +103,17 @@ def test_digitize_empty(self, x, bins, dtype):
         x_dp = dpnp.array(x)
         bins_dp = dpnp.array(bins)
 
+        # SAT-7822: w/a to avoid crash on Windows (observing on LNL)
+        # TODO: remove the w/a once resolved
+        if x_dp.size == 0 and is_win_platform():
+            device_mask = (
+                du.intel_device_info(x_dp.sycl_device).get("device_id", 0)
+                & 0xFF00
+            )
+            # upper byte of device_id for LNL architecture
+            if device_mask in [0x6400]:
```

Comment:
It'd be good to have a helper function which obtains a mask for the default or input device and checks if it matches with  specified device architecture.
While for LNL arch value it'd be better to expose it as a enum value.

Reply:
Meanwhile it might be helpful to add a [link](https://github.com/intel/compute-runtime/blob/master/shared/source/dll/devices/devices_base.inl) to the enum description on GitHub page where to find the values of device architectures.